source/features/index.tsx

```diff
@@ -104,12 +105,81 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 		document.head.append(<style>{options.customCSS}</style>);
 	}
 
+	const testedFeatures: [string[], string[]] | false | undefined = await cache.get('bisect');
+	if (testedFeatures !== undefined) {
+		if (testedFeatures === false) {
+			createMessageBox('Every feature has been disabled. If you still see the bug, try disabling the whole extension.', false);
+			return;
+		}
+
+		const [firstHalf] = testedFeatures;
+		options = {
+			...options,
+			...Object.fromEntries(__features__.map(feature => [`feature:${feature}`, firstHalf.includes(feature)]))
+		};
+
+		if (firstHalf.length === 0) {
+			createMessageBox('Every feature has been disabled. Can you see the bug?');
+			return;
```

Comment:
Didn't realize this at first but using `return` every time shouldn't work... yet it does :thinking:

Reply:
It works because it should, but it it's discarding the `log` setting below.

This whole diff should be its own function, like `Object.assign(options, bisectFeatures)`